roles/openshift-applier/tasks/process-one-entry.yml

```diff
@@ -54,7 +54,7 @@
   - item.namespace is defined
   - item.namespace|trim != ''
 
-- name: "Create OpenShift objects for '{{ entry.object }}' with params"
+- name: "Create OpenShift objects for '{{ item.name }}' with params"
```

Comment:
The `name` isn't guaranteed to be set, so this could potentially generate errors.